sdk/containerregistry/azure-containerregistry/samples/sample_upload_download_manifest.py

```diff
@@ -74,11 +73,23 @@ def upload_download_manifest(self):
             print(b"".join(download_manifest_stream))
             # Download the layers
             for layer in downloaded_manifest.layers:
-                with client.download_blob(repository_name, layer.digest).data as layer_stream:
-                    print(b"".join(layer_stream))
+                try:
+                    with client.download_blob(repository_name, layer.digest) as stream:
+                        with open("layer_file", "wb") as layer_file:
```

Comment:
This was just a quick comment sample I made, please use a more accurate file name for the sample.